dns_config/test/api_acl_test.go

```diff
@@ -7,87 +7,244 @@ Testing AclAPIService
 
 // Code generated by OpenAPI Generator (https://openapi-generator.tech);
 
-package dns_config
+package dns_config_test
 
 import (
+	"bytes"
 	"context"
+	"encoding/json"
+	"github.com/infobloxopen/bloxone-go-client/dns_config"
+	"github.com/infobloxopen/bloxone-go-client/internal"
+	openapiclient "github.com/infobloxopen/bloxone-go-client/keys"
+	"github.com/stretchr/testify/require"
+	"io"
+	"net/http"
 	"testing"
+)
 
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
+type RoundTripFunc func(req *http.Request) *http.Response
 
-	openapiclient "github.com/infobloxopen/bloxone-go-client/dns_config"
-	"github.com/infobloxopen/bloxone-go-client/internal"
-)
+func (f RoundTripFunc) RoundTrip(req *http.Request) (*http.Response, error) {
+	return f(req), nil
+}
 
-func Test_dns_config_AclAPIService(t *testing.T) {
+func NewTestClient(fn RoundTripFunc) *http.Client {
+	return &http.Client{
+		Transport: RoundTripFunc(fn),
+	}
+}
 
-	configuration := internal.NewConfiguration()
-	apiClient := openapiclient.NewAPIClient(configuration)
+func Test_dns_config_AclAPIService(t *testing.T) {
 
 	t.Run("Test AclAPIService AclCreate", func(t *testing.T) {
 
-		t.Skip("skip test") // remove to run test
+		//t.Skip("skip test") // remove to run test
 
-		resp, httpRes, err := apiClient.AclAPI.AclCreate(context.Background()).Execute()
+		dummyAcl := dns_config.ConfigACL{
+			Comment: openapiclient.PtrString("This is a dummy ACL for testing."),
+			Id:      openapiclient.PtrString("dummyAclId"),
+			Name:    "dummyAclName",
+		}
 
-		require.Nil(t, err)
-		require.NotNil(t, resp)
-		assert.Equal(t, 200, httpRes.StatusCode)
+		testClient := NewTestClient(func(req *http.Request) *http.Response {
+			require.Equal(t, "POST", req.Method)
```

Comment:
use the constant from `http` package for all methods and status 

```suggestion
			require.Equal(t, http.MethodPost, req.Method)
```